atom/browser/api/atom_api_dialog.cc

```diff
@@ -86,6 +87,23 @@ struct Converter<file_dialog::DialogSettings> {
 
 namespace {
 
+// Consider downloads 'dangerous' if they go to the home directory on Linux and
+// to the desktop on any platform.
+bool DownloadPathIsDangerous(const base::FilePath& download_path) {
```

Comment:
cc @jumde - this will conflict with your PR I think and the safe_browser changes should take priority

Reply:
in your branch please update this to use download_prefs.cc version of DownloadPathIsDangerous

Reply:
or remove the whole patch if this is no longer necessary

Reply:
@jumde please remove this changes in https://github.com/brave/muon/pull/409 once you rebased. Thank you

Reply:
same functionality is covered in https://chromium.googlesource.com/chromium/src/+/533c62725be56fe49be4ab548e7dd7061b1c977c/chrome/browser/download/download_prefs.cc#154